_episodes/08-putting-it-all-together.md

```diff
@@ -7,8 +7,11 @@ questions:
     - "Why should I use Python to create plots?"
 objectives:
     - "Import the pyplot toolbox to create figures in Python."
+    - "Use matplotlib to make further adjustments to Pandas or plotnine objects."
 keypoints:
-    - "FIXME"
+    - "Matplotlib is the engine behind plotnine and Pandas plots."
+    - "The object-oriented way of matplotlib makes detailed customization possible."
```

Comment:
I think keypoints have to be more specific `Plots created with Pandas and plotnine can be customized with matplotlib methods` ?

Reply:
As a replacement for the second key point? I like that.

Reply:
I like the suggestion as well